text/0000-i-am-become-death.md

```diff
@@ -0,0 +1,173 @@
+- Rust Issue: (leave this empty)
+
+
+# Summary
+
+Adds a proper path for APIs to migrate from unstable to stable, including a
+full shepherd and final comment period. Also establishes guidelines for
+how RFCs and PRs should be handled and when RFCs or PRs should be preferred.
+
+
+
+
+# Motivation
+
+Now that the trains and subteams have been running for a while, we have enough
+data to flesh out the details on the *Life and Death of an API*. So as to keep
+us on the same page, this RFC posits the following premises:
+
+* RFCs are heavyweight:
+    * RFCs generally take at minimum 2 weeks from posting to land. In
+      practice it can be more on the order of months for particularly
+      controversial changes.
+    * RFCs are a lot of effort to write; especially for non-native speakers or
+      for members of the community whose strengths are more technical than literary.
+    * RFCs may involve pre-RFCs and several rewrites to accommodate feedback.
+    * RFCs require a dedicated shepherd to herd the community and author towards
+      consensus.
+    * RFCs require review from a majority of the subteam, as well as an official
+      vote.
+    * RFCs can't be downgraded based on their complexity. Full process always applies.
+      Easy RFCs may certainly land faster, though.
+    * RFCs can be very abstract and hard to grok the consequences of (no implementation).
+
+* PRs are low *overhead* but potentially expensive nonetheless:
+    * Easy PRs can get insta-merged by any rust-lang contributor.
+    * Harder PRs can be easily escalated. You can ping subject-matter experts for second
+      opinions. Ping the whole team!
+    * Easier to grok the full consequences. Lots of tests and Crater to save the day.
+    * PRs can be accepted optimistically with bors, buildbot, and the trains to guard
+      us from major mistakes making it into stable. The size of the nightly community
+      at this point in time can still mean major community breakage regardless of trains,
+      however.
+    * HOWEVER: Big PRs can be a lot of work to make only to have that work rejected for
+      details that could have been hashed out first. *This is the motivation for
+      having RFCs*.
+
+* RFCs are *only* meaningful if a significant and diverse portion of the community actively
+  participates in them. The official teams are not sufficiently diverse to establish
+  meaningful community consensus by agreeing amongst themselves.
+
+* If there are *tons* of RFCs -- especially trivial ones -- people are less likely to
+  engage with them. Official team members are super busy. Domain experts and industry
+  professionals are super busy *and* have no responsibility to engage in RFCs. Since
+  these are *exactly* the most important people to get involved in the RFC process,
+  it is important that we be maximally friendly towards their needs. We cannot build
+  a robust language using only the opinion of hobbyists and students with all the time
```

Comment:
I think having all the time in the world and being a hobbyist or student are pretty much orthogonal.